scripts/globals/weaponskills.lua

```diff
@@ -959,3 +959,61 @@ function takeWeaponskillDamage(defender, attacker, params, finaldmg, slot, tpHit
 
     return finaldmg;
 end
+
+function applyAftermathEffect(player, tp, spower_1, spower_2, spower_3)
+    spower_1 = spower_1 or 1
+    spower_2 = spower_2 or 1
+    spower_3 = spower_3 or 1
+
+    -- AFTERMATH LEVEL 1
+
+    if ((tp >= 1000) and (tp <= 1100)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 10, 0, 180, 0, spower_1);
+    elseif ((tp >= 1101) and (tp <= 1200)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 11, 0, 180, 0, spower_1);
+    elseif ((tp >= 1201) and (tp <= 1300)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 12, 0, 180, 0, spower_1);
+    elseif ((tp >= 1301) and (tp <= 1400)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 13, 0, 180, 0, spower_1);
+    elseif ((tp >= 1401) and (tp <= 1500)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 14, 0, 180, 0, spower_1);
+    elseif ((tp >= 1501) and (tp <= 1600)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 15, 0, 180, 0, spower_1);
+    elseif ((tp >= 1601) and (tp <= 1700)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 16, 0, 180, 0, spower_1);
+    elseif ((tp >= 1701) and (tp <= 1800)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 17, 0, 180, 0, spower_1);
+    elseif ((tp >= 1801) and (tp <= 1900)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 18, 0, 180, 0, spower_1);
+    elseif ((tp >= 1901) and (tp <= 1999)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV1, 19, 0, 180, 0, spower_1);
+
+    -- AFTERMATH LEVEL 2
+
+    elseif ((tp >= 2000) and (tp <= 2100)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 24, 0, 180, 0, spower_2);    
+    elseif ((tp >= 2101) and (tp <= 2200)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 28, 0, 180, 0, spower_2);
+    elseif ((tp >= 2201) and (tp <= 2300)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 32, 0, 180, 0, spower_2);
+    elseif ((tp >= 2301) and (tp <= 2400)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 36, 0, 180, 0, spower_2);
+    elseif ((tp >= 2401) and (tp <= 2500)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 40, 0, 180, 0, spower_2);
+    elseif ((tp >= 2501) and (tp <= 2600)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 44, 0, 180, 0, spower_2);
+    elseif ((tp >= 2601) and (tp <= 2700)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 48, 0, 180, 0, spower_2);
+    elseif ((tp >= 2701) and (tp <= 2800)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 52, 0, 180, 0, spower_2);
+    elseif ((tp >= 2801) and (tp <= 2900)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 56, 0, 180, 0, spower_2);
+    elseif ((tp >= 2901) and (tp <= 2999)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV2, 59, 0, 180, 0, spower_2);
+
+    -- AFTERMATH LEVEL 3
+
+    elseif ((tp == 3000)) then
+	player:addStatusEffect(EFFECT_AFTERMATH_LV3, 45, 0, 120, 0, spower_3);
```

Comment:
btw, I don't use tabs.  We *hates* them precious.  So either something in one of my editors needs tweaking, or they're getting copied and pasted.